RELEASE/scripts/autoscend.ash

```diff
@@ -2409,170 +2419,114 @@ boolean LX_attemptPowerLevel()
 {
 	set_property("auto_powerLevelAdvCount", get_property("auto_powerLevelAdvCount").to_int() + 1);
 	set_property("auto_powerLevelLastAttempted", my_turncount());
-
+	set_property("auto_powerLevelLastLevel", my_level());
+	set_property("auto_newbieOverride", true);
```

Comment:
should not be setting `auto_newbieOverride` ever.

Reply:
Fair enough. it was already doing it in L13_powerlevel(), i just moved it here.
Its purpose is to allow it to spend more than 52 power leveling... which to be fair we probably do not want to do completely automatically. better to have the user switch it instead after 52 turns power leveling. So I will remove this.

Reply:
change made